CHANGELOG.md

```diff
@@ -4,6 +4,9 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [3.6.0] - 2025-03-12
+## Added
+- Added the `table.deletion.enabled` property to separate table deletion into its own configuration item.
```

Comment:
Maybe add: `Note: this changes behaviour before this release partitioned tables were deleted if all partitions expired, we consider this a bug so we switched the behaviour but made it possible to keep the old behaviour through this property.`